{
 "errors": {
  "importlib_metadata.docs": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmp5_msnfra/lib/python3.6/site-packages/importlib_metadata/docs/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmp5_msnfra/lib/python3.6/site-packages/importlib_metadata/docs/__init__.py'",
    ""
   ]
  },
  "importlib_metadata.docs.conf": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmp5_msnfra/lib/python3.6/site-packages/importlib_metadata/docs/conf.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmp5_msnfra/lib/python3.6/site-packages/importlib_metadata/docs/conf.py'",
    ""
   ]
  },
  "importlib_metadata.tests": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmp5_msnfra/lib/python3.6/site-packages/importlib_metadata/tests/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmp5_msnfra/lib/python3.6/site-packages/importlib_metadata/tests/__init__.py'",
    ""
   ]
  },
  "importlib_metadata.tests.data": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmp5_msnfra/lib/python3.6/site-packages/importlib_metadata/tests/data/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmp5_msnfra/lib/python3.6/site-packages/importlib_metadata/tests/data/__init__.py'",
    ""
   ]
  },
  "importlib_metadata.tests.test_main": {
   "exception": "The subprocess /usr/share/miniconda/envs/test/bin/python has crashed (EOFError('Ran out of input'), stderr=).",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 248, in _send",
    "    is_exception, traceback, result = pickle_load(self._get_process().stdout)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/_compatibility.py\", line 28, in pickle_load",
    "    return pickle.load(file)",
    "EOFError: Ran out of input",
    "",
    "During handling of the above exception, another exception occurred:",
    "",
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/helpers.py\", line 488, in wrapper",
    "    return func(self, line, column, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 213, in complete",
    "    return completion.complete()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 169, in complete",
    "    cached_name, completion_names = self._complete_python(leaf)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 311, in _complete_python",
    "    completion_names += self._complete_inherited(is_function=False)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/completion.py\", line 424, in _complete_inherited",
    "    next(filters)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/value/klass.py\", line 206, in get_filters",
    "    metaclasses = self.get_metaclasses()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/value/klass.py\", line 394, in get_metaclasses",
    "    for value in lazy_base.infer():",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/lazy_value.py\", line 48, in infer",
    "    return self.context.infer_node(self.data)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 222, in infer_node",
    "    return infer_node(self, node)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 156, in infer_node",
    "    return _infer_node_if_inferred(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 169, in _infer_node_if_inferred",
    "    return _infer_node_cached(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 174, in _infer_node_cached",
    "    return _infer_node(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/debug.py\", line 81, in wrapper",
    "    return func(*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 82, in wrapper",
    "    return func(context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 197, in _infer_node",
    "    value_set = context.infer_node(first_child)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 222, in infer_node",
    "    return infer_node(self, node)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 156, in infer_node",
    "    return _infer_node_if_inferred(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 169, in _infer_node_if_inferred",
    "    return _infer_node_cached(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 174, in _infer_node_cached",
    "    return _infer_node(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/debug.py\", line 81, in wrapper",
    "    return func(*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 82, in wrapper",
    "    return func(context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 184, in _infer_node",
    "    return infer_atom(context, element)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 304, in infer_atom",
    "    return context.py__getattribute__(atom, position=position)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 75, in py__getattribute__",
    "    values = ValueSet.from_sets(name.infer() for name in names)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/base_value.py\", line 426, in from_sets",
    "    for set_ in sets:",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/context.py\", line 75, in <genexpr>",
    "    values = ValueSet.from_sets(name.infer() for name in names)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/names.py\", line 284, in infer",
    "    self.tree_name",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/__init__.py\", line 21, in wrapper",
    "    return built_functions[public_name](*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/stdlib.py\", line 867, in wrapper",
    "    return func(inference_state, context, tree_name)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/django.py\", line 177, in wrapper",
    "    result = func(inference_state, context, tree_name)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/syntax_tree.py\", line 744, in tree_name_to_values",
    "    types = imports.infer_import(context, tree_name)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/cache.py\", line 44, in wrapper",
    "    rv = function(obj, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 53, in infer_import",
    "    _prepare_infer_import(module_context, tree_name)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 116, in _prepare_infer_import",
    "    return from_import_name, tuple(import_path), import_node.level, importer.follow()",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 297, in follow",
    "    self._inference_state, self.import_path, sys_path, self._module_context",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 383, in import_module_by_names",
    "    ) for parent_module_value in value_set",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 383, in <listcomp>",
    "    ) for parent_module_value in value_set",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/__init__.py\", line 21, in wrapper",
    "    return built_functions[public_name](*args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/plugins/flask.py\", line 20, in wrapper",
    "    return callback(inference_state, import_names, module_context, *args, **kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 119, in wrapper",
    "    for p in parent_module_values",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/base_value.py\", line 426, in from_sets",
    "    for set_ in sets:",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/gradual/typeshed.py\", line 119, in <genexpr>",
    "    for p in parent_module_values",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/imports.py\", line 434, in import_module",
    "    is_global_search=False,",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 133, in wrapper",
    "    kwargs=kwargs,",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 226, in run",
    "    return self._send(id(inference_state), function, args, kwargs)",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/inference/compiled/subprocess/__init__.py\", line 260, in _send",
    "    stderr,",
    "jedi.api.exceptions.InternalError: The subprocess /usr/share/miniconda/envs/test/bin/python has crashed (EOFError('Ran out of input'), stderr=).",
    ""
   ]
  }
 },
 "symbols": [
  "importlib_metadata.ConfigParser",
  "importlib_metadata.Distribution",
  "importlib_metadata.EntryPoint",
  "importlib_metadata.FastPath",
  "importlib_metadata.FileHash",
  "importlib_metadata.FileNotFoundError",
  "importlib_metadata.IsADirectoryError",
  "importlib_metadata.MetaPathFinder",
  "importlib_metadata.MetadataPathFinder",
  "importlib_metadata.ModuleNotFoundError",
  "importlib_metadata.NotADirectoryError",
  "importlib_metadata.NullFinder",
  "importlib_metadata.PackageNotFoundError",
  "importlib_metadata.PackagePath",
  "importlib_metadata.PathDistribution",
  "importlib_metadata.PermissionError",
  "importlib_metadata.Prepared",
  "importlib_metadata.PyPy_repr",
  "importlib_metadata.__all__",
  "importlib_metadata.__metaclass__",
  "importlib_metadata.__version__",
  "importlib_metadata._compat",
  "importlib_metadata._compat.FileNotFoundError",
  "importlib_metadata._compat.IsADirectoryError",
  "importlib_metadata._compat.ModuleNotFoundError",
  "importlib_metadata._compat.NotADirectoryError",
  "importlib_metadata._compat.NullFinder",
  "importlib_metadata._compat.PermissionError",
  "importlib_metadata._compat.PyPy_repr",
  "importlib_metadata._compat.__all__",
  "importlib_metadata._compat.__metaclass__",
  "importlib_metadata._compat.disable_stdlib_finder",
  "importlib_metadata._compat.email_message_from_string",
  "importlib_metadata._compat.install",
  "importlib_metadata._compat.map",
  "importlib_metadata._compat.py2_message_from_string",
  "importlib_metadata._compat.str",
  "importlib_metadata._compat.suppress",
  "importlib_metadata._compat.unique_everseen",
  "importlib_metadata._compat.unique_ordered",
  "importlib_metadata.distribution",
  "importlib_metadata.docs",
  "importlib_metadata.email_message_from_string",
  "importlib_metadata.entry_points",
  "importlib_metadata.files",
  "importlib_metadata.install",
  "importlib_metadata.map",
  "importlib_metadata.metadata",
  "importlib_metadata.pathlib",
  "importlib_metadata.requires",
  "importlib_metadata.str",
  "importlib_metadata.suppress",
  "importlib_metadata.tests",
  "importlib_metadata.tests.fixtures.DALS",
  "importlib_metadata.tests.fixtures.DistInfoPkg",
  "importlib_metadata.tests.fixtures.EggInfoFile",
  "importlib_metadata.tests.fixtures.EggInfoPkg",
  "importlib_metadata.tests.fixtures.FileBuilder",
  "importlib_metadata.tests.fixtures.Fixtures",
  "importlib_metadata.tests.fixtures.LocalPackage",
  "importlib_metadata.tests.fixtures.LocalPackage.files",
  "importlib_metadata.tests.fixtures.NullFinder",
  "importlib_metadata.tests.fixtures.OnSysPath",
  "importlib_metadata.tests.fixtures.SiteDir",
  "importlib_metadata.tests.fixtures.__metaclass__",
  "importlib_metadata.tests.fixtures.build_files",
  "importlib_metadata.tests.fixtures.install_finder",
  "importlib_metadata.tests.fixtures.save_cwd",
  "importlib_metadata.tests.fixtures.tempdir",
  "importlib_metadata.tests.test_api.APITests",
  "importlib_metadata.tests.test_api.OffSysPathTests",
  "importlib_metadata.tests.test_integration.FinderTests",
  "importlib_metadata.tests.test_integration.IntegrationTests",
  "importlib_metadata.tests.test_integration.LocalProjectTests",
  "importlib_metadata.tests.test_integration.files",
  "importlib_metadata.tests.test_zip.TestEgg",
  "importlib_metadata.tests.test_zip.TestZip",
  "importlib_metadata.unique_ordered",
  "importlib_metadata.version"
 ]
}